packages/server/src/db/index.js

```diff
@@ -440,11 +455,22 @@ function getInterestedCodes() {
 }
 
 async function getAgency(agencyId) {
-    const query = `SELECT id, name, abbreviation, parent, warning_threshold, danger_threshold 
-    FROM agencies WHERE id = ?;`;
-    const result = await knex.raw(query, agencyId);
+    const query = knex.select(
+        'id',
+        'name',
+        'abbreviation',
+        'parent',
+        'warning_threshold',
+        'danger_threshold',
+    )
+        .from(TABLES.agencies)
+        .where({
+            id: agencyId,
+        })
+        .leftJoin('tenants', 'tenants.main_agency_id', '=', `${TABLES.agencies}.main_agency_id`);
```

Comment:
I think this should join `agencies.tenant_id = tenants.id` instead of using `main_agency_id` like this, in case you are querying about a subagency.